src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -160,10 +160,10 @@ public bool HighEntropyVA
         /// <summary>
         /// Specifies the list of instrumentation kinds to be used during compilation.
         /// </summary>
-        public string InstrumentationKinds
+        public string Instrument
```

Comment:
This property is renamed "Instrument" to match the name of the command-line option.